components/next_steps_view/__snapshots__/next_steps_view.test.tsx.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`components/next_steps_view should match snapshot 1`] = `
+<div
+  className="app__content NextStepsView"
+  id="app-content"
+>
+  <div
+    className="NextStepsView__header"
+  >
+    <div
+      className="NextStepsView__header-headerText"
+    >
+      <div
+        className="NextStepsView__header-headerTopText"
+      >
+        <FormattedMessage
+          defaultMessage="Welcome to Mattermost"
+          id="next_steps_view.welcomeToMattermost"
+          values={Object {}}
+        />
+      </div>
+      <div
+        className="NextStepsView__header-headerBottomText"
+      >
+        <FormattedMessage
+          defaultMessage="Here are some recommended next steps to help you collaborate"
```

Comment:
This copy changed to "Here are some recommended next steps to help you get started"